### PR TITLE
Fixed error NU1101: Unable to find package

### PR DESCRIPTION
**Pull request by Mailaender**
as suggested in https://github.com/actions/virtual-environments/issues/1090#issuecomment-748452120

### CODE REVIEW

**Comment by pchote** [comment]
This is already included in #17989, fwiw.

**Comment by Mailaender** [comment]
That makes it somehow even more annoying.

**Comment by abcdefg30** [comment]
If https://github.com/OpenRA/OpenRA/pull/17989#issuecomment-751518711 is blocking, then we could merge this first.

**Review by abcdefg30 — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -37,6 +37,7 @@ jobs:
       - name: Check Code
         shell: powershell
         run: |
+          dotnet nuget locals all --clear
```
Comment (phrohdoh) [inline .github/workflows/ci.yaml:40]: IMO would be nice to have a comment explaining why we have to do this, but won't block approval on it

**Review by phrohdoh — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Mailaender) [reply .github/workflows/ci.yaml:40]: I don't really know why it is required. Assume some kind of overzealous caching on those Azure machines.

**Comment by reaperrr** [comment]
Needs rebase.

**Comment by pchote** [comment]
Changing the target branch to prep-2012 would allow us to merge it there directly. This is no longer needed on bleed.

**Comment by Mailaender** [comment]
Updated.

**Pull request merged**